pkg/components/node-problem-detector/template.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodeproblemdetector
+
+const chartValuesTmpl = `
+settings:
```

Comment:
We may want to bump the version to recently released 0.8.7. Also, please check if they support ARM and maybe multi-arch images. It would be cool.

Reply:
I am not able to check for multi-arch image support. `docker manifest inspect` gives an un-authorization error.

```sh
Get https://k8s.gcr.io/v2/node-problem-detector/node-problem-detector/manifests/v0.8.7: unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication
```

Reply:
We should update the version in our values, not in upstream chart assets.

About the multi-arch, it sucks that k8s.gcr.io does not allow you to check it :exploding_head: We can just try deploying it on ARM cluster and have an e2e test for it.

Reply:
@knrt10 image version is still overwritten in assets not in here.

Reply:
Ok, updating it. Got confused first. Thanks for checking

Reply:
What is the rationale behind updating the version to `v0.8.7` when the upstream chart hasn't changed it yet?

Reply:
To keep things up to date, to avoid shipping outdated software and to have important fixes applied.